### DISCUSSION TITLE
Windows 10 & 11 Build

### DISCUSSION THREAD

**Discussion by crazyideas21** [noise-lab/iot-inspector-client#199] [General]
How to download, install, and run IoT Inspector on Windows: https://github.com/nyu-mlab/iot-inspector-client/wiki/Download-&-Install

In this thread, talk about issues you encountered. We'd love to hear your feedback!

**Comment by crazyideas21** [comment]
If you encounter any issues, please describe here or email your issues to iot-inspector-study@nyu.edu

**Comment by rosstex** [comment]
Hello Danny! When will the Mac binary be released? :)

**Comment by crazyideas21** [comment]
@rosstex Hopefully by February 24? We're currently tackling this issue: https://github.com/nyu-mlab/iot-inspector-client/issues/224

**Comment by ygroenewald** [comment]
Hi Danny, I have installed IoT Inspector on my Windows 11 laptop and it shows 9 devices on the dashboard. However, all 9 devices comes up as "Unnamed Device ....". Am I doing something wrong or something that I am not doing? Also, it has been running for a while now, but it never updates the "Analyzing the data; ..." blocks under each device, e.g.

![image](https://github.com/nyu-mlab/iot-inspector-client/assets/127762069/e54a138f-6a48-4243-b19a-748779384cb9)

I also got this error about 40 minutes after IoT Inspector started running ...

![image](https://github.com/nyu-mlab/iot-inspector-client/assets/127762069/8c1b71d1-b460-4085-8f0e-74779ec3fc77)

**Comment by crazyideas21** [comment]
Hi @ygroenewald 

All devices are unnamed at the beginning. Sorry for not making it clearer. Will clarify in the upcoming release.

The above issue is a known bug related to `streamlit`. Will fix in the next version.

**Comment by AndrewQuijano** [comment]
As of IoT Inspector 3, a Windows executable will be generated for a cleaner installation process.
